doc/changes/changes_0.5.4.md

```diff
@@ -0,0 +1,25 @@
+# Extension Manager 0.5.4, released 2023-10-18
+
+Code name: Migrate to Exasol 8
+
+## Summary
+
+Starting with this release, Extension Manager requires Exasol 8. Also integration tests for extension definitions must run with Exasol 8.
+
+To skip tests on non v8 versions, you can use the following new method:
+
+```java
+ExasolVersionCheck.assumeExasolVersion8(exasolTestSetup)
```

Comment:
Should we mention the required import statement here?

`import com.exasol.extensionmanager.itest;`

Reply:
I have another question.
AFAIK integration tests for most extensions (e.g. virtual schemas)  are using a docker DB.
Which defines the database version up-front by providing the docker image name.
Isn't a check based on internal database tables somewhat redundant then?

Reply:
I propose to add the description above to the developer guide, too.
Please add a note how version is detected (by SQL query) either to java doc or developer guide.